documentation/templates/bolt_cmdlet_reference.md.erb

```diff
@@ -19,9 +19,22 @@ These PowerShell cmdlets are available in the Bolt PowerShell module.
 
 <%= command[:description] %>
 
-| Parameter | Description | Mandatory |
-| --- | --- | :-: |
-<% command[:options].each do |option| -%>
-| `-<%= option[:name] -%><%= option[:arg] %>` | <%= option[:help_msg].gsub("<", "&lt;").split("\n").join('<br>') %> | <%= option[:mandatory] ? '✓' : '' %>
-<% end %>
-<% end %>
+<table class="table">
+  <thead>
+   <tr>
+        <th>Option</th>
+        <th>Description</th>
+        <th align="center">Mandatory?</th>
+    </tr>
+  </thead>
+  <tbody>
+   <% command[:options].each do |option| -%>
+      <tr>
+        <td>`-<%= option[:name] -%><%= option[:arg] %>`</td>
```

Comment:
```suggestion
        <td>-<%= option[:name] -%><%= option[:arg] %></td>
```

Looks like using HTML tables means the codeblocks won't be generated.

Reply:
Oooh, I missed that. We have to replace the backticks with <code>. I'll fix that